mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/MesosSchedulerClient.java

```diff
@@ -71,7 +75,7 @@ public MesosSchedulerClient(
     }
 
     @NotNull
-    public Observable<Protos.Event> openEventStream(@NotNull final Call subscription) {
+    public Observable<In> openEventStream(@NotNull final Out subscription) {
```

Comment:
I was thinking about it in terms of the client: `In` is the message that the client receives from Mesos, while `Out` is the message that the client sends to Mesos.

What would be a good way to name these type parameters unambiguously? `FromMesos` and `ToMesos`?